README.md

```diff
@@ -100,7 +114,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema defines what columns the data tables will have.  We relate a one to many relationship by using a foreign key.
 ```
```

Comment:
is it just what columns? how does it know which tables to update the columns in? It defines tables rather than just columns, but it does that too.